Apps/GatewayApi/test/unit/Services.Test/RegistrationServiceTests.cs

```diff
@@ -482,23 +485,15 @@ private static CreateUserProfileMock SetupCreateUserProfileMock(
 
             return new(
                 service,
-                jobServiceMock,
-                createUserRequest,
-                DateTime.UtcNow,
-                jwtEmailAddress);
+                jobServiceMock);
         }
 
-        private static CreateUserProfileExceptionMock SetupCreateUserProfileThrowsExceptionMock(
+        private static IRegistrationService SetupCreateUserProfileThrowsExceptionMock(
```

Comment:
Method name no longer corresponds to the returned type.  I suggest something like `SetupRegistrationServiceForCreateUserProfileThrowsException()`.

Reply:
done